challenges/largest-product/largest_array.py

```diff
@@ -0,0 +1,17 @@
+def LargestArray(arr):
+    
+"""
+Finds the largest product in a 2D array
+"""
```

Comment:
The doc string must be indented as well or your code will break due to a SyntaxError.